.travis.yml

```diff
@@ -2,6 +2,8 @@ language: rust
 matrix:
   include:
     - rust: 1.27.2
+      script:
+        - cargo test --all-targets
```

Comment:
What is this for exactly? As far as I understand, this overwrites the global `script: cargo test`, yes? But `cargo test --all-targets` does more than just `cargo test`, right?

Reply:
cargo test --all-targets is the equivalent to --lib --bins --tests --benches --examples which will exclude --doc (see https://doc.rust-lang.org/cargo/commands/cargo-test.html)  The code in src/lib.rs does not compile in 2015 version (Rust 1.27) so I am excluding doctests.  I think --no-doc would make more sense, but the flag is --all-targets